velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -403,6 +403,58 @@ TEST_F(DecimalArithmeticTest, decimalDivDifferentTypes) {
        makeFlatVector<int64_t>({100, 200, -300, 400}, DECIMAL(12, 2))});
 }
 
+TEST_F(DecimalArithmeticTest, decimalMod) {
+  // short % short -> short.
```

Comment:
There are 6 signatures being registered, but only 4 of them are tested. Let's add tests for all signatures.